src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderPreferences/GeneralType.php

```diff
@@ -99,6 +99,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'label' => $this->trans('Recalculate shipping costs after editing the order', 'Admin.Shopparameters.Feature'),
                 'help' => $this->trans('Automatically updates the shipping costs when you edit an order.', 'Admin.Shopparameters.Help'),
+            ])
+            ->add('autoremove_not_valid_cartrules', SwitchType::class, [
```

Comment:
hello !

If you are making a feature flag, your field should be added in `FeatureFlagsType.php`  , and then it will appear in Configure > Advanced Parameters > Experimental Features

Reply:
@matthieu-rolland It's not a featue flag, just a settings in Prestashop. However, I think it should be hardcoded.